src/test/TxTests.cpp

```diff
@@ -112,7 +112,7 @@ void
 checkTransaction(TransactionFrame& txFrame, Application& app)
 {
     REQUIRE(txFrame.getResult().feeCharged ==
-            app.getLedgerManager().getTxFee());
+            app.getLedgerManager().getTxFee() * txFrame.getOperations().size());
```

Comment:
I don't understand the change here. Can you explain @avi-kik ?

Reply:
The original code assumed one operation per tx.